index.html

```diff
@@ -1062,7 +1128,8 @@ <h1 class="sub-title">Contact Me</h1>
 
             </div>
             <div class="contact-right">
-                <form name="submit-to-google-sheet">
+                <h2>Any Queries?</h2>
+                <form name="submit-to-google-sheet" class="contact-form">

```

Comment:
Could you make the form method to be post instead of get ? right now when we click submit it shows all the data in the address bar.